package.json

```diff
@@ -50,6 +50,7 @@
     "@angular/core": "14.1.3",
     "@angular/forms": "14.1.3",
     "@angular/material": "14.1.3",
+    "@angular/material-date-fns-adapter": "14.1.3",
     "@angular/material-moment-adapter": "14.1.3",
```

Comment:
do we need removing the moment one? or it's still needed?

Reply:
I'm not sure. I don't see usages of the moment adapter in ACA anymore, but there would still be its usages in ADF. There are usages of the base moment library in some ACA E2E test cases though